message_forwarder.go

```diff
@@ -56,7 +57,7 @@ func extractUUID(msg string) (string, error) {
 	contextRegexp := regexp.MustCompile(uuidContextRegexp)
 	validRegexp := regexp.MustCompile(uuidValidRegexp)
 	uuidContext := contextRegexp.FindString(msg)
-	uuid := validRegexp.FindString(uuidContext)
+	uuid := strings.ToLower(validRegexp.FindString(uuidContext))
```

Comment:
it was a PR for that
https://github.com/Financial-Times/coco-kafka-bridge/pull/24/files